docs/portal/image-registry/managing-registry-projects.md

```diff
@@ -0,0 +1,18 @@
+# Managing Registry Projects
+
+A registry project contains image repositories. You can use a project to group repositories that have a similar purpose, such as repositories for the same application or for the same organization. These groupings also allow you to manage access control for your projects.
+
+##Registry Projects Page
```

Comment:
A blankspace is missing after ##. As a result, the heading is not rendered. 

Tip: You might want to preview a page to verify after changes are made to the page.

Reply:
Fixed.